services/web/server/src/simcore_service_webserver/director_v2_core.py

```diff
@@ -96,27 +100,27 @@ async def _request_director_v2(
     **kwargs,
 ) -> DataBody:
 
-    session = get_client_session(app)
     try:
-        async with session.request(
-            method, url, headers=headers, json=data, **kwargs
-        ) as response:
-            payload: Union[Dict, str] = (
-                await response.json()
-                if response.content_type == "application/json"
-                else await response.text()
-            )
-
-            # NOTE:
-            # - `sometimes director-v0` (via redirects) replies
-            #   in plain text and this is considered an error
-            # - `director-v2` and `director-v0` can reply with 204 no content
-            if response.status != expected_status.status_code or (
-                response.status != web.HTTPNoContent and isinstance(payload, str)
-            ):
-                raise DirectorServiceError(response.status, reason=str(payload))
-
-            return payload
+        async for attempt in AsyncRetrying(**DEFAULT_RETRY_POLICY):
+            with attempt:
```

Comment:
so the request retry is applied to all incoming request

Reply:
yes and the syntax was anyway wrong before. with unbound variables. But still I don't think that thing is necessary.

Reply:
I kept it so that whatever the idea was, is kept.